core/src/main/java/nl/nn/adapterframework/http/IbisWebServiceSender.java

```diff
@@ -40,6 +40,7 @@
  * @since 4.2
  */
 public class IbisWebServiceSender implements ISender, HasPhysicalDestination {
+	private ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
Please let this class extend SenderBase, and inherit name and classloader from that class